src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,61 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { Tab } from '../types/Tabs';
+
+type Props = {
+  tabs: Tab[]
+  selectedTabId: string
+  onTabSelected: (tab: Tab) => void
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId);
+  const handleTabClick = (tab: Tab) => {
+    return () => tab.id !== selectedTab?.id && onTabSelected(tab);
+  };
```

Comment:
what will you do if selectedTab will be "undefined"?
maybe it's better to have a backup plan?)
```suggestion
  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
  const handleTabClick = (tab: Tab) => {
    return () => tab.id !== selectedTab.id && onTabSelected(tab);
  };
```

Reply:
it was no doubt that something went bad ) thanks